WordPressLint/src/main/java/org/wordpress/android/lint/WordPressIssueRegistry.kt

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.lint
+
+import com.android.tools.lint.client.api.IssueRegistry
+import com.android.tools.lint.client.api.Vendor
+import com.android.tools.lint.detector.api.CURRENT_API
+import org.wordpress.android.lint.WordPressAndroidImportInViewModelDetector.Companion.ISSUE_ANDROID_IMPORT_IN_VIEWMODEL
+
+class WordPressIssueRegistry : IssueRegistry() {
+    override val api = CURRENT_API
+    override val minApi = MIN_API
+    override val issues get() = listOf(
+            WordPressRtlCodeDetector.SET_PADDING,
+            WordPressRtlCodeDetector.SET_MARGIN,
+            WordPressRtlCodeDetector.GET_PADDING,
+            ISSUE_ANDROID_IMPORT_IN_VIEWMODEL,
+    )
+    override val vendor = Vendor(
+            vendorName = "WordPress Android",
+            feedbackUrl = "https://github.com/wordpress-mobile/WordPress-Lint-Android/issues",
+            identifier = "org.wordpress.android.lint",
+    )
+    companion object {
+        const val MIN_API = 10
```

Comment:
I'm not sure how low this can go, but I had to set it lower than `CURRENT_API`, since we are not on the latest within our other projects yet.

Reply:
This is the exact mapping that I used to make the determination. We could _in theory_ go lower, but I only tried it as low as we needed for WordPress-Android and WordPress-FluxC-Android. This is an important consideration for the future, though, if we want to be good stewards of a public facing library project.